src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,118 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.task.Event;
+import duke.task.Deadline;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Would it be better to divide the import statement into different parts to avoid big chunks?